docs/content/annotations.md

```diff
@@ -54,13 +54,13 @@ supported are:
 
 * `rule` - applies to the individual rule statement (within the same file). Default, when metadata block precedes rule.
 * `document` - applies to all of the rules with the same name in the same package (across multiple files)
-* `package` - applies to all of the rules in the package (within the same file). Default, when metadata block precedes package.
+* `package` - applies to all of the rules in the package (across multiple files). Default, when metadata block precedes package.
 * `subpackages` - applies to all of the rules in the package and all subpackages (recursively, across multiple files)
 
 Since the `document` scope annotation applies to all rules with the same name in the same package 
-and the `subpackages` scope annotation applies to all packages with a matching path, metadata blocks with 
+and the `packages` and `subpackages` scope annotations apply to all packages with a matching path, metadata blocks with 
 these scopes are applied over all files with applicable package- and rule paths. 
-As there is no ordering across files in the same package, the `document` and `subpackages` scope annotations 
+As there is no ordering across files in the same package, the `document`, `scope`, and `subpackages` scope annotations 
```

Comment:
Nit: Do we mean `package` instead of `scope`?

Reply:
d'oh! 🤦‍♂️